try section revisions? (#134)
greenelab/iscb-diversity-manuscript: content/20.results.md

We downloaded the metadata of manuscripts published in these journals from PubMed, which provided 176,110 articles for evaluation.
For each article, we extracted its last author's fore name and last name for analysis.

### Defining metrics and a field-specific background

We were faced with a number of choices as we set out to examine representation within the field.
Here, we outline the choices that we made and the rationale for these choices.

For one step, we needed to determine whether the most appropriate unit for analysis was researchers or honor and authorship events.
We elected to perform the analysis at the level of honors and authorships, not individuals.
Our rationale was that, if a scientist was honored three times, these three honoree slots represented distinct selection processes and should be considered separately.

> We also elected to use authorships instead of authors because name disambiguation can be error prone, and because the overall design of the work was to examine distributions the precise linking of researchers was not necessary to address the research question.

Reviewer: ```suggestion
We also elected to use authorships instead of authors because name disambiguation can be error prone, and because the overall design of the work was to examine authorship distributions, the precise linking of researchers was not necessary to address the research question.
```

Practical options included selecting authors in key society-endorsed journals or selecting those based on some article metadata.
We elected to perform the analysis to all 176,110 PubMed articles with a computational biology MeSH term to generate the largest possible set of relevant literature.

We assumed that, among the authors of a specific paper, the senior author (often research advisors) would be most likely to be invited for a keynote or honored as a fellow.
Based on field-specific conventions, we could have selected the last or corresponding author.
We found that corresponding authors were often first and/or last suggesting that they were highly involved with more corresponding authors being last authors than first authors.
Given the convention and the observation that corresponding authors were more often last, we selected the last author as the most appropriate in this setting.

We present the results with the selected strategy (Fig. {@fig:workflow}); however, we performed this work through multiple iterations varying these parameters to examine the extent to which they influenced the results.

> We performed the analysis across different combinations of the above choices and results were consistent with respect to the broad conclusions, though the numerical results differed (see Methods and supplementary figure S3).

Reviewer: remove repetitive "perform" and "results" from previous sentence.
```suggestion
Our findings across different combinations of the above choices were consistent with respect to the broad conclusions, though the numerical results differed (see Methods and supplementary figure S3).
```
